### PR TITLE
docs: edited pyenv set up

### PR DESCRIPTION
**Pull request by christinewang5**
## Summary
Edited pyenv set up to match the [pyenv docs](https://github.com/pyenv/pyenv#basic-github-checkout)

### CODE REVIEW

**Comment by electronjoe** [comment]
Thanks for the docs fixup @christinewang5!

**Review by electronjoe — APPROVED** [review]

File: docs/docusaurus/versioned_docs/version-1.0.1/basics/prerequisites.md [hunk 1]
```diff
@@ -26,8 +26,12 @@ brew install pyenv
 
 # Replace .zshrc with your appropriate shell RC file
 # IMPORTANT: Use .bash_profile, not .bashrc for bash
```
Comment (hcgatewood) [inline docs/docusaurus/versioned_docs/version-1.0.1/basics/prerequisites.md]:
```suggestion
# NOTE: this assumes you're using zsh.
# See the above pyenv install instructions if using alternative shells.
```

File: docs/docusaurus/versioned_docs/version-1.0.1/basics/prerequisites.md [hunk 2]
```diff
@@ -26,8 +26,12 @@ brew install pyenv
...
-echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc
+echo 'eval "$(pyenv init --path)"' >> ~/.zprofile
+echo 'eval "$(pyenv init -)"' >> ~/.zshrc
 exec $SHELL
+
+# Restart your terminal
```
Comment (hcgatewood) [inline docs/docusaurus/versioned_docs/version-1.0.1/basics/prerequisites.md]:
```suggestion
# IMPORTANT: close your terminal tab and open a new one before continuing
```

**Review by hcgatewood — APPROVED** [review]
Thanks @christinewang5! Msg me when ready and I'll merge 💯

File: docs/docusaurus/versioned_docs/version-1.0.1/basics/prerequisites.md [hunk 3]
```diff
@@ -24,10 +24,14 @@ the [pyenv installation instructions](https://github.com/pyenv/pyenv#installatio
 ```bash
 brew install pyenv
 
-# Replace .zshrc with your appropriate shell RC file
-# IMPORTANT: Use .bash_profile, not .bashrc for bash
-echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc
+# NOTE: this assumes you're using zsh.
```
Comment (themarwhal) [inline docs/docusaurus/versioned_docs/version-1.0.1/basics/prerequisites.md]: do we want to update this to the non-versioned doc as well? Or do we only want to update it for v1.0.1

Comment (christinewang5) [reply docs/docusaurus/versioned_docs/version-1.0.1/basics/prerequisites.md]: you are so right, I didn't notice there were versions, but will edit this right now... 🤦🏻‍♀️

File: docs/readmes/basics/prerequisites.md [hunk 4]
```diff
@@ -17,18 +17,22 @@ Install the following tools:
...
+echo 'eval "$(pyenv init --path)"' >> ~/.zprofile
+echo 'eval "$(pyenv init -)"' >> ~/.zshrc
 exec $SHELL
-pyenv install 3.8.10
-pyenv global 3.8.10
+
+# IMPORTANT: close your terminal tab and open a new one before continuing
+
+pyenv install 3.7.3
+pyenv global 3.7.3
```
Comment (hcgatewood) [inline docs/readmes/basics/prerequisites.md]: This should still be 3.8.10 if I'm not mistaken

**Review by hcgatewood — CHANGES_REQUESTED** [review]

**Review by hcgatewood — APPROVED** [review]
💯🔥

**Pull request merged**